pkg/api/v1beta3/types.go

```diff
@@ -828,6 +828,15 @@ type NodeStatus struct {
 	Conditions []NodeCondition `json:"conditions,omitempty"`
 }
 
+// NodeInfo is the information collected on the node.
+type NodeInfo struct {
+	TypeMeta `json:",inline"`
+	// The number of cores in this machine.
+	NumCores int `json:"numCores"`
```

Comment:
Cores and memory can be consolidated inside NodeSpec but I am planning to return more information (I have another pull here on cadvisor: https://github.com/google/cadvisor/pull/560).

(And as far as I understood Capacity is currently not updated with the information from the host)

Anyway I think it's just a matter of deciding where to return this information (spec/info), I mentioned it in a previous comment.